src/main/frontend/handler/editor.cljs

```diff
@@ -339,7 +339,7 @@
   [block value]
   (if (and (state/enable-timetracking?)
            (not= (:block/content block) value))
-    (let [new-marker (first (util/safe-re-find marker/bare-marker-pattern (or value "")))
+    (let [new-marker (first (util/safe-re-find marker/marker-pattern (or value "")))
```

Comment:
Breaks heading-style TODOs:

<img width="279" alt="image" src="https://user-images.githubusercontent.com/72891/155649241-b39b5b91-4aed-4681-98e2-afff7bb86b54.png">

Reply:
fixed in c250ca7f4076280afce3ae7613a2754b4c26e04e